azext_edge/tests/edge/orchestration/resources/test_schema_registry_int.py

```diff
@@ -113,12 +117,7 @@ def test_schema_registry_lifecycle(settings_with_rg, tracked_resources):
     assert alt_registry_name in list_registry_names
 
     # DELETE
-    try:
-        run(f"az iot ops schema registry delete -n {registry_name} -g {registry_rg} -y")
-    except CLIInternalError as e:
-        # delete does not return correct status code - remove once fixed
-        if "ERROR: Operation returned an invalid status 'OK'" not in e.error_msg:
-            raise e
+    run(f"az iot ops schema registry delete -n {registry_name} -g {registry_rg} -y")
```

Comment:
are we no longer having the deletion error anymore? saw the try catch has been removed